Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,21 @@
+namespace Trxlog2Html.Extensions;
+public static class StringExtensions
+{
+    public static bool IsImage(this string value)
+    {
+        return value.EndsWith(".png");
+    }
+    public static bool IsVideo(this string value)
+    {
+        return value.EndsWith(".webm");
```

Comment:
Similar to IsImage, we might want to consider additional video extensions when determining if the file is a video. Also, using StringComparison.OrdinalIgnoreCase for case-insensitive checks could be beneficial.